src/views/virtualmachines/actions/tests/useVirtualMachineActionsProvider.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { cleanup } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+
+import useVirtualMachineActionsProvider from '../hooks/useVirtualMachineActionsProvider';
+
+import { exampleRunningVirtualMachine } from './mocks';
+
+jest.mock('@openshift-console/dynamic-plugin-sdk', () => ({
+  useK8sWatchResource: jest.fn(() => [[], true]),
+}));
+afterEach(cleanup);
+
+describe('useVirtualMachineActionsProvider tests', () => {
+  it('Test running VM actions', () => {
+    const { result } = renderHook(() =>
+      useVirtualMachineActionsProvider(exampleRunningVirtualMachine),
+    );
+    expect(result.current).toMatchSnapshot();
+
+    const actions = [...result.current.actions];
```

Comment:
no need for `[...result.current.actions]`
just write `result.current.actions.find((a) => a.id === 'vm-action-start');`